tools/ogstools/Extract/ExtractSurface.xml

```diff
@@ -0,0 +1,68 @@
+<tool id="extractSurface" name="ogstools_extractSurface" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@" profile="@PROFILE@">
```

Comment:
- `id` should be all lower case
- my suggestion for `name` would be `OGS ExtractSurface` (in particular I would like to avoid `_`)